app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -22,7 +24,26 @@ def self.event_monitor_class
 
   def create_project(project)
     connect.create_project_request(project)
-  rescue KubeException => e
-    raise MiqException::MiqProvisionError, "Unexpected Exception while creating project: #{e}"
+  end
+
+  def users_from_provider
+    connect.get_users
+  end
+
+  def user_from_provider(user_name)
+    connect.get_user(user_name)
+  end
+
+  def user_exists_in_provider?(user_name)
```

Comment:
@zakiva Yes, this will raise an exception.  @simon3z recommended I remove exceptions from these methods.  It is then the caller's responsibility to handle them.

However, if it is the consensus that we handle the exceptions here, then that's fine.  I'll follow the team's recommendation.